src/System.Private.ServiceModel/src/System/ServiceModel/NetHttpsBinding.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ServiceModel.Channels;
+using System.Xml;
+
+namespace System.ServiceModel
+{
+    public class NetHttpsBinding : HttpBindingBase
+    {
+        BinaryMessageEncodingBindingElement _binaryMessageEncodingBindingElement;
+        NetHttpMessageEncoding _messageEncoding;
+        BasicHttpsSecurity _basicHttpsSecurity;
+
+        public NetHttpsBinding() : this(BasicHttpsSecurity.DefaultMode) { }
+
+        public NetHttpsBinding(BasicHttpsSecurityMode securityMode)
+        {
+            if (securityMode == BasicHttpsSecurityMode.TransportWithMessageCredential)
+            {
+                throw ExceptionHelper.PlatformNotSupported(SR.Format(SR.UnsupportedSecuritySetting, "securityMode", securityMode));
+            }
+
+            Initialize();
+            _basicHttpsSecurity.Mode = securityMode;
+        }
+
+        [ComponentModel.DefaultValue(NetHttpMessageEncoding.Binary)]
```

Comment:
Do we still need these [ComponentModel.DefaultValues] around in the implementation file? I thought it's only for config purposes when in the implementation file, but these should be specified in the ref assembly now, right?